File: 2974-minimum-number-game/2974-minimum-number-game.go
func numberGame(nums []int) []int {
    result := make([]int, 0, len(nums))
    sort.Ints(nums)

    for i := 0; i < len(nums); i += 2 {
        result = append(result, nums[i + 1], nums[i])
    }

    return result
}

File: 2974-minimum-number-game/2974-minimum-number-game.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var numberGame = function(nums) {
    let arr = nums.sort((a,b) => a-b);
    for(let i=0;i<nums.length-1; i=i+2){
        let temp = arr[i]
        arr[i] = arr[i+1]
        arr[i+1] = temp
    }
    
    return arr;
};